src/tasks/level10.ts

```diff
@@ -139,22 +140,24 @@ export const GiantQuest: Quest = {
       completed: () => have($item`S.O.C.K.`),
       do: $location`The Penultimate Fantasy Airship`,
       choices: () => {
-        return { 178: 2, 182: have($item`model airship`) ? 1 : 4 };
+        return { 178: 2 };
       },
       post: () => {
         if (have($effect`Temporary Amnesia`)) cliExecute("uneffect Temporary Amnesia");
       },
       orbtargets: () => [],
       outfit: () => {
-        if (get("_saberForceUses") < 5 && !have($item`Mohawk wig`)) {
+        if (get("_saberForceUses") < 5 && !have($item`Mohawk wig`) && !have($item`bat wings`)) {
```

Comment:
The equip: bat wings code here is redundant, but it's not a typo. We're trying to avoid incidental saber usage while we have bat wings equipped. Saber and Bat Wings don't play nice.

Reply:
No I still don't understand it.
`if (weDoNotHaveBatWings) { equip(batWings); }`

Reply:
The block of code can literally only run if you do not have bat wings, in which case it's a bit useless to ask for, or deny bat wings. As it's no longer relevant.